curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -7,21 +7,25 @@ forumTopicId: 301838
 ---
 
 ## Description
+
 <section id='description'>
 
 Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
 
 <div style='text-align: center;'>1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...</div>
 
-By considering the terms in the Fibonacci sequence whose values do not exceed <code>n</code>, find the sum of the even-valued terms.
+Find the sum of the even numbers in the Fibonacci sequence such that the numbers are less than or equal to <code>n</code>
```

Comment:
```suggestion
Find the sum of the even numbers in the Fibonacci sequence of numbers less than or equal to <code>n</code>
```
Does this read a little more clearly, perhaps?

Reply:
Sure thing